src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -35,6 +35,16 @@ interface CustomerInstitutionMetadataFormProps {
   editMode: boolean;
 }
 
+function inactiveFromIsInThePast(inactiveFrom: string) {
+  const now = new Date();
+  const inactiveFromDate = new Date(inactiveFrom);
+  return inactiveFromDate < now;
+}
+
+function isInactive(inactiveFrom: string | null | undefined): boolean {
```

Comment:
siden den nå er tatt vekk fra typen
```suggestion
function isInactive(inactiveFrom: string | undefined): boolean {
```